tests/foreman/virtwho/test_ui_virtwho.py

```diff
@@ -626,3 +626,67 @@ def test_positive_virtwho_manager_role(session, test_name, form_data):
         # Delete the created user
         session.user.delete(username)
         assert not session.user.search(username)
+
+
+def test_positive_overview_label_name(form_data, session):
+    """Verify the label name on virt-who config Overview Page.
+
+    :id: 68422042-648c-4652-a52b-5bf42462d2ae
+
+    :BZ: 1649928
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
+    """
+    name = gen_string('alpha')
+    form_data['name'] = name
+    hypervisor_type = form_data['hypervisor_type']
+    form_data['proxy'] = 'test.rexample.com:3128'
```

Comment:
a little typo, it's not rexample, it should be example:)

Reply:
oh, copy from test_positive_proxy_option, I will modify that as well